charts/posthog/values.yaml

```diff
@@ -869,6 +869,8 @@ zookeeper:
 clickhouse:
   # -- Whether to install clickhouse. If false, `clickhouse.host` must be set
   enabled: true
+  # -- Whether to install `clickhouse` CRDs.
+  installCRDs: false
```

Comment:
Worth changing this to `true` by default to avoid changing current behaviour, especially in a patch release.

fyi there are a few test [failures in CI](https://github.com/PostHog/charts-clickhouse/runs/8303362148?check_suite_focus=true), let us know if you need nay help running these tests locally.